### PR TITLE
ipa_config: add user and group ojectclasses parameters

### PR DESCRIPTION
**Pull request by SJay3**
##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ipa_config.py

### CODE REVIEW

File: plugins/modules/ipa_config.py [hunk 1]
```diff
@@ -40,6 +40,12 @@
     aliases: ["primarygroup"]
     type: str
     version_added: '2.5.0'
+  ipagroupobjectclasses:
+    description: A list of group objectclasses.
+    aliases: ["groupobjectclasses"]
+    type: list
+    elements: str
+    version_added: '7.2.0'
```
Comment (felixfontein) [inline plugins/modules/ipa_config.py:48]: 7.2.0 already has been released:
```suggestion
    version_added: '7.3.0'
```

File: plugins/modules/ipa_config.py [hunk 2]
```diff
@@ -91,6 +97,12 @@
     type: list
     elements: str
     version_added: '2.5.0'
+  ipauserobjectclasses:
+    description: A list of user objectclasses.
+    aliases: ["userobjectclasses"]
+    type: list
+    elements: str
+    version_added: '7.2.0'
```
Comment (felixfontein) [inline plugins/modules/ipa_config.py:105]:
```suggestion
    version_added: '7.3.0'
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by SJay3** [comment]
@felixfontein Thanks for your help. I fixed version and added changelog fragments

File: changelogs/fragments/7019-ipa_config-user-and-group-objectclasses.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ipa_config - add module parameters to manage freeipa user and group objectclasses
```
Comment (felixfontein) [inline changelogs/fragments/7019-ipa_config-user-and-group-objectclasses.yml:2]:
```suggestion
  - ipa_config - add module parameters to manage FreeIPA user and group objectclasses (https://github.com/ansible-collections/community.general/pull/7019).
```

**Comment by SJay3** [comment]
@felixfontein All fixes done. Need an approve

**Comment by felixfontein** [comment]
Looks good to me. I'll merge if nobody objects by the end of this week.

**Comment by SJay3** [comment]
@felixfontein LGTM

**Comment by felixfontein** [comment]
@SJay3 thanks for your contribution!

**Pull request merged**